revenue/revenueapp/templates/individual_venue.html

```diff
@@ -14,7 +14,7 @@ <h4><a href="{% url 'review_create' venue.id %}">Review this venue</a></h4>
 <img src="{{ venue.image }}" width="740px" class="center">
 
 <br>
-
+<h1 id="Individual_venue_heading_text">User review</h1>
```

Comment:
Can you pluralize this as the site has been changed to use multiple reviews and multiple users? It currently shows up on the page even if there are no reviews, fyi. Changing it to 'User Reviews' could work as a header for an empty part of the page.

Reply:
![image](https://user-images.githubusercontent.com/13088830/177655271-0bfbf78c-bda4-4400-8f9f-d914dedbae72.png)
This is what I am talking about: there is no user review on this page and there **could** be many reviews.